projects/packages/jetpack-mu-wpcom/src/features/wpcom-command-palette/wpcom-command-palette.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * WPCom Command Palette support for WordPress.com sites.
+ *
+ * @package automattic/jetpack-mu-plugins
+ */
+
+/**
+ * Check if the WPCom Command Palette should be loaded.
+ *
+ * @return bool
+ */
+function should_load_wpcom_command_palette() {
+	global $pagenow;
+	$excluded_pages = array(
+		'post.php',
+		'post-new.php',
+		'site-editor.php',
+	);
+	return isset( $pagenow ) && ! in_array( $pagenow, $excluded_pages, true );
+}
+
+/**
+ * Get the WPCom Command Palette JS configuration as a string.
+ *
+ * @return string
+ */
+function get_wpcom_command_palette_config_js() {
+	$host         = new Automattic\Jetpack\Status\Host();
+	$data         = array(
+		'siteId'       => Jetpack_Options::get_option( 'id' ),
+		'isAdmin'      => current_user_can( 'manage_options' ),
+		'isAtomic'     => $host->is_woa_site(),
+		'isSimple'     => $host->is_wpcom_simple(),
+		'isSelfHosted' => ! $host->is_wpcom_platform(),
+	);
+	$encoded_data = wp_json_encode( $data );
+
+	return "var commandPaletteConfig = $encoded_data;";
+}
+
+/**
+ * Load the WPCom Command Palette.
+ */
+function wpcom_load_command_palette() {
+	if ( ! should_load_wpcom_command_palette() ) {
+		return;
+	}
+
+	$command_palette_js_handle = 'command-palette-script';
+
+	wp_enqueue_script(
+		'command-palette-script',
+		'//widgets.wp.com/command-palette/build.min.js',
+		array(),
+		'1.0.1',
```

Comment:
We need a dynamic version here, otherwise we'll be always loading a cached version of the script which will ignore future changes.

For the `wpcom-block-editor` scripts we use a version that changes every 24 hours, so in the worst case scenario changes will need 24 hours to be visible on Atomic sites.

https://github.com/Automattic/jetpack/blob/3cf4ae640be8cf9cb53f37e8445bed7b62519a53/projects/plugins/jetpack/modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php#L318